evap/results/exporters.py

```diff
@@ -185,11 +192,13 @@ def writen(exporter, label="", style_name="default"):
     exporter.row += 1
     writec(exporter, label, style_name)
 
+
 def writec(exporter, label, style_name, rows=1, cols=1):
     """Write the cell in the next column of the current line."""
     _write(exporter, label, exporter.styles[style_name], rows, cols)
     exporter.col += 1
 
+
```

Comment:
methods in classes. i think we do have two newlines outside. https://www.python.org/dev/peps/pep-0008/#blank-lines